cli/ping.go

```diff
@@ -207,3 +222,12 @@ func (r *RootCmd) ping() *serpent.Command {
 	}
 	return cmd
 }
+
+func isAWSIP(awsRanges *cliutil.AWSIPRanges, ni *tailcfg.NetInfo) bool {
+	checkIP := func(ipStr string) bool {
+		ip, err := netip.ParseAddr(ipStr)
+		return err == nil && awsRanges.CheckIP(ip)
+	}
+
+	return checkIP(ni.GlobalV4) || checkIP(ni.GlobalV6)
```

Comment:
I wouldn't use an anonymous function for this, just do the parsing and checking twice in a row

Reply:
Make sure to check if it's not empty